### PR TITLE
Loss Operations: Move Reduction under losses namespace

### PR DESCRIPTION
**Pull request by ManrajGrover**
This PR moves `Reduction` to losses namespace to align with Tensorflow

https://www.tensorflow.org/api_docs/python/tf/losses/Reduction

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-core/1011)

### CODE REVIEW

**Comment by ManrajGrover** [comment]
Hi Daniel,

I agree this might break code for users. Just to be clear, will we be incorporating this change when major release happens since some users might expect it to be under `tf.losses`?

**Comment by ManrajGrover** [comment]
I think we can close this.

**Comment by dsmilkov** [comment]
Thanks for the understanding
